static/styles/subscriptions.css

```diff
@@ -320,16 +320,16 @@ h4.stream_setting_subsection_title {
             color: inherit;
         }
     }
+
+    .form-inline {
+        margin-bottom: 0;
+    }
```

Comment:
For reference -> https://github.com/zulip/zulip/pull/21264/commits/e1496103d1405bafdd9c33264106f1d6d7e1d634#r816250236